host/bcm283x/gpio.go

```diff
@@ -325,13 +328,18 @@ func (p *Pin) FastOut(l gpio.Level) {
 //
 // PWM pins
 //
-// PWM0 is exposed on pins 12, 18 and 40.
+// PWM0 is exposed on pins 12, 18 and 40. However, PWM0 is used for generating
+// clock for DMA and unavailable for PWM.
+// TODO(simokawa): Use other DREQ source like PCM.
 //
 // PWM1 is exposed on pins 13, 19, 41 and 45.
 //
 // PWM0 and PWM1 share the same 25Mhz clock source. The period must be a
```

Comment:
This is irrelevant to mention PWM0 if PWM0 cannot be used anymore.

Reply:
done